_docs_integrate/request-one-time-consent-of-peer.md

```diff
@@ -25,88 +25,107 @@ required_by:
 {: .notice--info}
 
-## Create the Request
+As there is already a Relationship between the Sender and the Recipient, the Sender can send the [Request via a Message]({% link _docs_integrate/requests-via-messages.md %}) to the Recipient. Even if the Relationship has already been established, the Sender could also send the [Request via a RelationshipTemplate]({% link _docs_integrate/requests-via-relationshiptemplates.md %}) to the Recipient, but this is not discussed further below.
 
-To create the Request you have to call the `POST /api/v2/Requests/Outgoing` route on the Sender Connector. Use the following JSON in the Request body:
+Please note that if there is no active Relationship between the Sender and the Recipient, the Request for one-time consent must be sent via a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) instead of a [Message]({% link _docs_integrate/data-model-overview.md %}#message). The process of [establishing a Relationship]({% link _docs_integrate/establish-relationships.md %}) is then initiated at the same time.
+{: .notice--info}
+
+### Create the Request
+
+As already indicated, the Request contains a [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem) within its `items` property.
+To create the Request, the Sender must proceed as described in the [Create outgoing Request]({% link _docs_use-cases/use-case-consumption-create-outgoing-request.md %}) use case, specifying a suitable payload as given in the example below.
+In this example, the value of the `mustBeAccepted` property of the ConsentRequestItem is set to `true`.
+Please note that the `<...>` notation is used as a placeholder for the actual data as usual and that the `link` property of the [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem) is optional and can therefore also be omitted.
 
 ```jsonc
 {
   "content": {
-    // the content property of the payload in the step before
+    "items": [
+      {
+        "@type": "ConsentRequestItem",
+        "mustBeAccepted": true,
+        "consent": "<issue that the Sender wants the Recipient to agree to>",
+        "link": "<link to external website with more information on the issue>"
+      }
+    ]
   },
-  "peer": "<the address of the Recipient Connector>"
+  "peer": "<Address of Recipient>"
 }
 ```
 
-Note that the Request is currently in status `Draft`.
+Before creating the Request, the Sender should check whether it is valid. This can be done by proceeding as described in the documentation of the [Check if outgoing Request can be created]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}) use case. The advantage of checking the validity of the Request first before attempting to create it is that the Sender would receive a more precise [error]({% link _docs_integrate/error-codes.md %}) description in the case of a faulty Request.
```

Comment:
The advantage of checking the validity of the Request first before attempting to create it is that the Sender **will** receive a more precise [error](http://localhost:4000/integrate/error-codes) description in the case of a faulty Request.